rosbridge_server/scripts/rosbridge_websocket_client.py

```diff
@@ -18,7 +20,7 @@
 
 ws = None
 protocol = None
-
+authenticate = False
```

Comment:
Shouldn't this variable be called 'user_auth' instead, which is then accessed as a global?